# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysvinit/musl.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sysvinit-2.88dsf/src/init.c.vanilla	2016-12-23 20:04:50.466561628 +0000
+++ sysvinit-2.88dsf/src/init.c	2016-12-23 20:05:24.006562204 +0000
@@ -73,6 +73,8 @@
 #  endif
 #endif
 
+#include <sys/ttydefaults.h>
+
 #include "init.h"
 #include "initreq.h"
 #include "paths.h"
